Build fails with nodemcu 3.0

When following the build docs I get a build failure of the following:

```
CC app/user/user_main.c
user_main.c:81:42: error: 'NODEMCU_EAGLEROM_PARTITION' undeclared here (not in a function)
     { NODEMCU_PARTITION_EAGLEROM,         0x00000,     0x0B000},
                                          ^
user_main.c:85:42: error: 'NODEMCU_IROM0TEXT_PARTITION' undeclared here (not in a function)
     { NODEMCU_PARTITION_IROM0TEXT,        0x10000,      0x0000},
                                          ^
user_main.c:86:42: error: 'NODEMCU_LFS0_PARTITION' undeclared here (not in a function)
     { NODEMCU_PARTITION_LFS,              0x0,          LUA_FLASH_STORE},
                                          ^
user_main.c:87:42: error: 'NODEMCU_SPIFFS0_PARTITION' undeclared here (not in a function)
     { NODEMCU_PARTITION_SPIFFS,           0x0,          SPIFFS_MAX_FILESYSTEM_SIZE},
                                          ^
user_main.c: In function 'nodemcu_init':
user_main.c:302:5: error: implicit declaration of function 'dbg_printf' [-Werror=implicit-function-declaration]
     NODE_ERR("\n");
     ^
cc1: all warnings being treated as errors
../../Makefile:422: recipe for target '.output/eagle/debug/obj/user_main.o' failed
```

This seems to be caused by the recent 3.0 "release" of nodemcu. Checking out the latest 2.x release in the nodemcu dir using `git checkout 2.2.1-master_20190405` fixes this.

--- a/README.md
+++ b/README.md
@@ -48,6 +48,10 @@ Konnected leverages the [NodeMCU](https://github.com/nodemcu/nodemcu-firmware) c
         git clone https://github.com/konnected-io/konnected-security.git
         git clone https://github.com/nodemcu/nodemcu-firmware.git
 
+1. Check out the 2.2.1 release of nodemcu
+
+        pushd nodemcu-firmware && git checkout 2.2.1-master_20190405 && popd
+
 1. Use the build-firmware script to kick off the build - providing a semantic version command line argument as shown below. The build script will automatically pull down the nodeMCU docker builder, build the base firmware, create an LFS image, and build a SPIFFS file system containing the entire Konnected application.
 
         cd konnected-security

